src/pretalx/agenda/templates/agenda/speaker.html

```diff
@@ -0,0 +1,38 @@
+{% extends "agenda/base.html" %}
+{% load i18n %}
+
+{% block content %}
+<h3>
+    {{ speaker.nick }}
+    {% if speaker.name %}
+        <small>{{ speaker.name }}</small>
+    {% endif %}
+</h3>
+
+<div class="row">
+    <div class="col-lg-8 col-md-6 col-xs-12">
+        {% if speaker.biography %}
+            {{ speaker.biography }}
+        {% else %}
+            {% trans 'This speaker did not provide a biography yet.' %}
```

Comment:
This could probably also say "$SpeakersNickname (or $SpeakersName?) did not provide a biography yet.", which sounds a little nicer. What do you think?